code/server/sv_ccmds.c

```diff
@@ -1297,11 +1298,11 @@ static void SV_ConTell_f(void) {
 	}
 
 	if ( Cmd_Argc() < 3 ) {
-		Com_Printf ("Usage: tell <client number> <text>\n");
+		Com_Printf ("Usage: tell <player> <text>\n");
```

Comment:
If both do the same it makes no sense to keep the 2 commands. /tell covers both numeric player handle and player name, so indeed we could alias /sayto to /tell (there is no /sayto in iourbanterror, as far as I can say)